test/unit/node/util.test.ts

```diff
@@ -198,10 +143,11 @@ describe("isHashMatch", () => {
     expect(async () => await util.isHashMatch(password, _hash)).not.toThrow()
```

Comment:
Looks like this should be updated as well (false instead of throw).

Reply:
Nice catch!